src/services/draw/drawn-feature.ts

```diff
@@ -50,6 +50,15 @@ export class DrawnFeature extends Feature {
     }
   }
 
+  public get featureStyle() {
+    return this._featureStyle
+  }
+
+  public set featureStyle(featureStyle: DrawnFeatureStyle) {
+    this._featureStyle = featureStyle
+    this.changed()
+  }
+
```

Comment:
Nice refactoring, thanks ! I'm just not a big fan of the "private" underscore syntax (reminding me of v3) as we've not been using it consistently in v4.

Reply:
ok, I see, what other prefix would you use ? If I write a getter for a non-private variable, I get a type error.

Reply:
Ah right, got it, I missed the getter/setter having the same names and that a prefix is actually needed here. Then the underscore makes perfectly sense to me.